--- v86d-0.1.3/Makefile.orig	2007-12-03 01:27:11.000000000 +0100
+++ v86d-0.1.3/Makefile	2008-01-26 21:10:11.424451532 +0100
@@ -13,17 +13,11 @@
 CFLAGS += -I$(KDIR)/include
 
 ifeq ($(call config_opt,CONFIG_X86EMU),true)
-	CFLAGS += -Ilibs/x86emu
-	LDFLAGS += -Llibs/x86emu
 	LDLIBS += -lx86emu
 	V86OBJS = v86_x86emu.o v86_mem.o
-	V86LIB = x86emu
 else
-	CFLAGS += -Ilibs/lrmi-0.10
-	LDFLAGS += -Llibs/lrmi-0.10 -static
 	LDLIBS += -llrmi
 	V86OBJS = v86_lrmi.o
-	V86LIB = lrmi
 endif
 
 DEBUG_BUILD =
