--- v86d-0.1.5.2/Makefile.orig	2008-07-21 19:10:25.248758520 +0200
+++ v86d-0.1.5.2/Makefile	2008-07-21 19:15:12.152748463 +0200
@@ -12,17 +12,12 @@
 CFLAGS ?= -Wall $(OPTFLAGS)
 
 ifeq ($(call config_opt,CONFIG_X86EMU),true)
-	CFLAGS += -Ilibs/x86emu
-	LDFLAGS += -Llibs/x86emu
 	LDLIBS += -lx86emu
 	V86OBJS = v86_x86emu.o v86_mem.o
-	V86LIB = x86emu
 else
-	CFLAGS += -Ilibs/lrmi-0.10
-	LDFLAGS += -Llibs/lrmi-0.10 -static -Wl,-z,execheap
+	LDFLAGS += -Wl,-z,execheap
 	LDLIBS += -llrmi
 	V86OBJS = v86_lrmi.o
-	V86LIB = lrmi
 endif
 
 DEBUG_BUILD =
