--- v86d-0.1.9/Makefile	2008-10-18 20:39:22.000000000 +0000
+++ v86d-0.1.9/Makefile	2008-10-18 20:41:41.000000000 +0000
@@ -9,20 +9,14 @@
 	CFLAGS += -I/usr/include/klibc
 endif
 
-CFLAGS ?= -Wall -g -O2
+CFLAGS ?= -Wall $(OPTFLAGS)
 
 ifeq ($(call config_opt,CONFIG_X86EMU),true)
-	CFLAGS += -Ilibs/x86emu
-	LDFLAGS += -Llibs/x86emu
 	LDLIBS += -lx86emu
 	V86OBJS = v86_x86emu.o v86_mem.o v86_common.o 
-	V86LIB = x86emu
 else
-	CFLAGS += -Ilibs/lrmi-0.10
-	LDFLAGS += -Llibs/lrmi-0.10 -static -Wl,--section-start,vm86_ret=0x9000
 	LDLIBS += -llrmi
 	V86OBJS = v86_lrmi.o v86_common.o
-	V86LIB = lrmi
 endif
 
 DEBUG_BUILD =
